pkg/config/config.go

```diff
@@ -498,6 +499,7 @@ func DefaultConfig() (*Config, error) {
 			DefaultSysctls:           []string{},
 			DefaultUlimits:           []string{},
 			AdditionalDevices:        []string{},
+			HooksDir:                 []string{hooks.DefaultDir},
```

Comment:
Do we need to add documentation somewhere else like in the template.go?

Reply:
we shouldn't. the template.go gets the hooks configured and generates based on that.